frontend/src/advising/ManageStudents.tsx

```diff
@@ -282,62 +337,139 @@ const StudentPreview = ({ id }: StudentPreviewProps) => {
       dispatch(resetUserAction());
     };
   }, []);
+
+  if (viewMode === "overview") {
```

Comment:
So just based on the figma, the expanded schedule views didn't need to have the search bar visible above it, so that part still could be a separate page like you had mentioned before with the student id and plan id. I think it's still fine like this but it might be nice to pull that part out into a separate page since it can be

Reply:
Yeah I did notice this, and I decided to just leave the search bar in because it doesn't hurt. Once we move the Single Student View and this page to their own separate routes, we can do what we feel is best with the search bar on those pages.